lms/djangoapps/grades/signals/handlers.py

```diff
@@ -148,6 +148,11 @@ def score_published_handler(sender, block, user, raw_earned, raw_possible, only_
 
     if update_score:
         # Set the problem score in CSM.
+
+        # adding temporary log for understanding EDUCATOR-1931
+        if user.is_anonymous():
+            log.info("Problem [%s] was accessed by anonymous user", block.location)
```

Comment:
do you want this to be a `log.exception` instead so we get the stack trace?

Reply:
This is only to check if current user is Anonymous user. Exception will be logged any way as it is getting logged currently.